util/seeds.js

```diff
@@ -78,10 +82,6 @@ class ScopedSeed
       include = [/.*/];
       break;
 
```

Comment:
should we be removing the none case?

Reply:
yeah, wasn't very clear before, now updated in README and the error message to print the valid values for scopeType.